view/frontend/templates/instant/wrapper.phtml

```diff
@@ -23,44 +24,26 @@
                     <div class="col-md-12">
                         <div>
                             {{#second_bar}}
-                            <div id="instant-search-bar-container">
-                                <div id="instant-search-box">
-                                    <div class="instant-search-bar-label">
-                                        <span><?php echo $block->escapeHtml(__('Current search')); ?></span>
-                                    </div>
-                                    <div class="instant-search-bar-wrapper">
-                                        <label for="instant-search-bar">
-                                            <?php echo $block->escapeHtml(__('Search :')); ?>
-                                        </label>
-
-                                        <input placeholder="<?php echo $block->escapeHtml(__('Search for products')) ?>"
-                                               id="instant-search-bar"
-                                               type="text"
-                                               autocomplete="off"
-                                               spellcheck="false"
-                                               autocorrect="off"
-                                               autocapitalize="off" />
-                                    </div>
-                                </div>
-                            </div>
+                               <div id="instant-search-bar"></div>
                             {{/second_bar}}
                         </div>
                     </div>
                 </div>
                 <div class="row">
-                    <div>
+                    <div class="col-md-12">
                         <div itemscope itemtype="http://schema.org/ItemList" class="hits">
-                            <div class="infos">
+                            <div class="algolia-infos">
                                 <div class="pull-left" id="algolia-stats"></div>
                                 <div class="pull-right">
                                     <div class="sort-by-label pull-left">
-                                        <?php echo $block->escapeHtml(__('SORT BY')); ?>
+                                        <?php echo $block->escapeHtml(__('SORT BY')); ?> &nbsp;
                                     </div>
                                     <div class="pull-left" id="algolia-sorts"></div>
                                 </div>
                                 <div class="algolia-clearfix"></div>
                             </div>
                             <div id="algolia-banner"></div>
+                            <div id="instant-empty-results-container"></div>
```

Comment:
this is the mounting point for the custom Suggestions widget 
Previously it was simply using the mounting point of the hits widget, which I found to be unnecessarily confusing, so I split them.

Reply:
why instant here & not Algolia as prefix?

Reply:
I wanted to keep the same naming as the previous ` <div id="instant-search-results-container"></div>`
So I could have but then some will ask the other question: why `algolia` an not `instant` prefix 😄  

Like you, I definitely think naming here deserves a big make over. I left that in the refactoring step which will come later.